# velocity_data.msg

int64 left_wheel_velocity
int64 right_wheel_velocity